Formula/hydra.rb

```diff
@@ -27,9 +28,11 @@ def install
     inreplace "configure" do |s|
       # Link against our OpenSSL
       # https://github.com/vanhauser-thc/thc-hydra/issues/80
-      s.gsub! "/opt/local/lib", Formula["openssl@1.1"].opt_lib
-      s.gsub! "/opt/local/*ssl", Formula["openssl@1.1"].opt_lib
-      s.gsub! "/opt/*ssl/include", Formula["openssl@1.1"].opt_include
+      s.gsub!(/^SSL_PATH=""$/, "SSL_PATH=#{Formula["openssl@1.1"].opt_lib}")
+      s.gsub!(/^SSL_IPATH=""$/, "SSL_IPATH=#{Formula["openssl@1.1"].opt_include}")
```

Comment:
```suggestion
      s.gsub!(/^SSL_IPATH=""$/, "SSL_IPATH=#{Formula["openssl@1.1"].opt_include}")
      s.gsub!(/^SSLNEW=""$/, "SSL_NEW=YES")
```
Technically any non-empty value should be fine.

Reply:
Ah, wait. Just tested it. This doesn't work...

Reply:
AH. There's a typo 😅

Reply:
Huh. This is one _weird_ configure script.

Reply:
Yea I don't really understand it either. It seems our setting of the `SSL_` variables early skips the check for whether OpenSSL is new enough, which causes the script to assume that OpenSSL is old.

Why that didn't happen on macOS (I checked), I don't know. Maybe it happened anyway but superenv linker magic fixed it anyway.